server/src/main/java/org/elasticsearch/common/Rounding.java

```diff
@@ -739,21 +739,15 @@ public final long nextRoundingValue(long utcMillis) {
 
     static class TimeIntervalRounding extends Rounding {
         static final byte ID = 2;
-        /** Since, there is no offset of -1 ms, it is safe to use -1 for non-fixed timezones */
-        private static final long TZ_OFFSET_NON_FIXED = -1;
 
         private final long interval;
         private final ZoneId timeZone;
-        /** For fixed offset timezones, this is the offset in milliseconds, otherwise TZ_OFFSET_NON_FIXED */
-        private final long fixedOffsetMillis;
```

Comment:
We don't need this any more because we can just ask the lookup if it is fixed across the bounds we need.